zone/mob.cpp

```diff
@@ -8726,27 +8740,24 @@ bool Mob::CheckLosCheat(Mob* other) {
 	return true;
 }
 
-bool Mob::CheckLosCheatExempt(Mob* other)
```

Comment:
This is all commented out and just returns false ? Should we just remove this code ?